ui/src/timeMachine/components/fluxFunctionsToolbar/ToolbarFunction.tsx

```diff
@@ -37,7 +39,11 @@ class ToolbarFunction extends PureComponent<Props, State> {
         onMouseLeave={this.handleStopHover}
         data-testid={testID}
       >
-        {this.tooltip}
+        {isActive && (
```

Comment:
The variables list in the script editor needed the same style of tooltips as the functions list, so I extracted them to a component called `BoxTooltip`.


<img width="513" alt="Screen Shot 2019-03-18 at 8 58 13 AM" src="https://user-images.githubusercontent.com/638955/54544445-e340bb80-495c-11e9-8a1d-f9b6042cf3a5.png">
<img width="823" alt="Screen Shot 2019-03-18 at 8 58 34 AM" src="https://user-images.githubusercontent.com/638955/54544450-e6d44280-495c-11e9-847c-95c4803e97b9.png">